### PR TITLE
StepsProgress component

### PR DESCRIPTION
**Pull request by Nap1400**

### CODE REVIEW

File: src/components/stepsProgress/StepsProgress.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+
+export const StepsProggress = ({currentStep = 1, totalSteps}) => {
+    if (totalSteps === null || totalSteps === undefined){
+        throw console.error('Value for totalSteps required');
```
Comment (raynesmike) [inline src/components/stepsProgress/StepsProgress.js:6]: lets also check for totalSteps=0

**Review by raynesmike — APPROVED** [review]

File: src/components/stepsProgress/StepsProgress.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+
+export const StepsProggress = ({currentStep = 1, totalSteps}) => {
+    if (totalSteps === null || totalSteps === undefined){
```
Comment (Dulaya) [inline src/components/stepsProgress/StepsProgress.js:5]: I think this checks for more condition: `if(!totalSteps && !Number.isInteger(totalSteps))`

**Comment by Dulaya** [comment]
Also are you not using styling at all?

**Review by Dulaya — CHANGES_REQUESTED** [review]

**Review by Dulaya — APPROVED** [review]
Went over with Nicholas on video call to get issues fixed. Works fine.

**Review by lasharela — APPROVED** [review]

**Pull request merged**